docs/source/continuous-deployment.rst

```diff
@@ -0,0 +1,26 @@
+Continuous Deployment
+======================
+We utilize OpenShift to deploy our Sync2Jira instance. Here we assume you have configured your quay.io repo to push to some sort of message bus. This will likely have to be done with an external script (something we do not support).
+
+The following environmental variables will have to be set:
+
+Related to OpenShift:
+    1. :code:`TOKEN` :: OpenShift Token to be used
+    2. :code:`ENDPOINT` :: OpenShift Endpoint to use
+    3. :code:`NAMESPACE` :: OpenShift Namespace to use
+Message Bus Related:
+    1. :code:`CERT` :: Cert file that should be used (can be in .pem format)
+    2. :code:`KEY` :: Key file that should be used (in .key format)
+    3. :code:`CA_CERTS` :: CA Certs that should be used
+    4. :code:`UMB_QUERY` :: Query that we should be using
```

Comment:
Instead of `UMB_...` call this `ACTIVEMQ_...`

UMB is the codename for our internal instance of AcitveMQ.